client/src/sections/Fridge/Header/Navbar/Navbar.tsx

```diff
@@ -3,19 +3,33 @@ import {useHistory} from 'react-router-dom';
 import {Button, ButtonGroup} from '@shopify/polaris';
 
 import './Navbar.css';
+import {useDispatch} from "react-redux";
+import {logout} from "../../../../store/users/action";
```

Comment:
I added aliases to the base folders, avoid having more than 2 directory "up"s.

`import {logout} from 'store/users/action;`